### ISSUE TITLE
[Eclipse-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.4.7][Eclipse][Wipro]:Keyboard focus is not visible for Help button in "Available Software" window 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1532]
**Repro Steps:**

1. Open and Start Eclipse.
2. Press "Alt+H" to open Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4.  In Available Software dialog , navigate to Work with text edit box and provide http://dl.microsoft.com/eclipse/ link and hit enter
5. Then tab up to Tab  up to "Contact all update sites during install to find required software" checkbox then press tab and observe the keyboard focus

 This issue is observed across all the windows in Install window.(Install Dialog and Review License Dialog)

**Expected Result:**
Keyboard focus should be visible for Help Button

**Actual Result:**
Keyboard focus is not visible for Help Button

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version**: Version 1709(OS Build 16299.192)
![keyboard focus is not visible for help button](https://user-images.githubusercontent.com/22956130/36205259-2b04ef68-11b4-11e8-8219-13ff500601dc.jpg)

**Comment by Eskibear** [comment]
It's part of the Eclipse IDE UI, we cannot modify that.

**Comment by Eskibear** [comment]
and it's out of the toolkit's scope

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749988
